### ISSUE TITLE
Error running 'go build main.go' with go-chrome called

### ISSUE THREAD

**Issue by j4ng5y** [mkenney/go-chrome#132]
I was trying to implement go-chrome in a scraping package I am building. Everything was going well until I tried to build the package with `go build`.

Go Version:
`go version go1.12 linux/amd64`

Output of go env: 
```bash
GOARCH="amd64"
GOBIN=""
GOCACHE="<REDACTED>/.cache/go-build"
GOEXE=""
GOFLAGS=""
GOHOSTARCH="amd64"
GOHOSTOS="linux"
GOOS="linux"
GOPATH="<REDACTED>/go"
GOPROXY="direct"
GORACE=""
GOROOT="/usr/local/go"
GOTMPDIR=""
GOTOOLDIR="/usr/local/go/pkg/tool/linux_amd64"
GCCGO="gccgo"
CC="gcc"
CXX="g++"
CGO_ENABLED="1"
GOMOD="<REDACTED>/go.mod"
CGO_CFLAGS="-g -O2"
CGO_CPPFLAGS=""
CGO_CXXFLAGS="-g -O2"
CGO_FFLAGS="-g -O2"
CGO_LDFLAGS="-g -O2"
PKG_CONFIG="pkg-config"
GOGCCFLAGS="-fPIC -m64 -pthread -fmessage-length=0 -fdebug-prefix-map=/tmp/go-build448021466=/tmp/go-build -gno-record-gcc-switches"
```

### What version of `go-chrome` are you using (tag, hash, etc.)?
v1.0.0-rc6

### What behavior do you expect? What are you trying to accomplish?
Build my package with `go build`

### What behavior are you experiencing instead?
```bash
# github.com/mkenney/go-chrome/tot/socket
../../../../pkg/mod/github.com/mkenney/go-chrome@v1.0.0-rc6/tot/socket/socket.socketer.go:239:24: cannot call pointer method on err.("github.com/bdlm/errors".Err)
../../../../pkg/mod/github.com/mkenney/go-chrome@v1.0.0-rc6/tot/socket/socket.socketer.go:239:24: cannot take the address of err.("github.com/bdlm/errors".Err)
```

### How can this behavior be reproduced?
1) Use VGO (go.mod/go.sum)
2) Implement an example from _examples
3) build the package via `go build`

**Comment by mkenney** [comment]
Hi, thanks for reporting that, I'll take a look now.

**Comment by mkenney** [comment]
@j4ng5y I was able to replicate that, it's strange that there isn't a compiler error when building with vendored dependencies. I don't know how to explain that. I fixed the reference though and it's building for me (I was using `go` instead of `vgo` but I expect it's the same result).

The fixes are in [v1.0.0-rc7](https://github.com/mkenney/go-chrome/releases/tag/v1.0.0-rc7), please let me know if that solves the issue for you.

**Comment by mkenney** [comment]
Closing, but feel free to re-open if this is still an issue for you.
